### ISSUE TITLE
ONIX technique for List 196 Code 99

### ISSUE THREAD

**Issue by rickj** [w3c/publ-a11y#468]
In a conversation with a publisher, they were struggling with understanding how their information in https://ns.editeur.org/onix/en/196/99 would be displayed.

In looking at https://www.editeur.org/files/ONIX%203/APPNOTE%20Accessibility%20metadata%20in%20ONIX%20(advanced).pdf at the top of page 28 it provides no guidance.

In looking at https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/#conformance-group I see no technique for displaying this information.

**Comment by gautierchomel** [comment]
I agree this information should be shown, even if used only a few times today. 

I am unsure if the correct place is in Additional Accessibility Information or if we should consider adding it to the Accessibility Summary section as it relates to the metadata pieces where the Publishing organisation speaks directly to the consumer.

**Comment by rickj** [comment]
Checking back on this, as I have publishers looking for clarity.

**Comment by chrisONIX** [comment]
@rickj What kind of guidelines are you looking for in ONIX guidelines?  
We can add examples of how to use this in ONIX document but we can not make a recommendation for how to display it.
Plus some text about what this is for?  Let us know what would be useful in the ONIX guidelines. 

We have published guidelines link to the EU GPSR requirements linked to contact information which has to be displayed on websites that sell goods to consumers in Europe and the sort of displays we are seeing for physical products would probably be the same for digital products and that maybe appropriate for this discussion as it is a very similar process.

https://www.editeur.org/files/ONIX%203/APPNOTE%20Product%20safety%20requirements%20in%20ONIX.pdf

**Comment by GeorgeKerscher** [comment]
It does not look like schema has a field for this. I believe it is a best practice for publishers to have an accessibility area that is linked from the home page. On this accessibility page there should be contact inform where issues can be reported.

There is always the accessibility summary where the publisher can put their contact information. If the community feels that this should be a recommendation, we could put it in the example in the accessibility summary.

**Comment by rickj** [comment]
I think that is the recommendation I'd like to have.  If there is content in this ONIX field, it should be appended to the Accessibility Summary

**Comment by GeorgeKerscher** [comment]
I like this approach! So if ONIX has this information, it could immediately followthe accessibility summary or it could be concatinated. In the case where the publisher also puts this information in the EPUB schema accessibility summary itself, it would appear twice, which I do not think is too bad.

**Comment by rickj** [comment]
Did this get missed?

**Comment by gregoriopellegrino** [comment]
I'm working on it :)
